packages/wallet/ui/src/util/WalletBackendAdapter.js

```diff
@@ -175,6 +170,33 @@ export const makeWalletBridgeFromFollowers = (
     notifierKits.purses.updater.updateState(harden(purses));
   };
 
+  const signSpendAction = async data => {
+    const {
+      signers: { interactiveSigner },
+    } = keplrConnection;
+    if (!interactiveSigner) {
+      throw new Error(
+        'Cannot sign a transaction in read only mode, connect to keplr.',
+      );
+    }
+    return interactiveSigner.submitSpendAction(data);
+  };
+
+  const getNotifierMethods = Object.fromEntries(
+    Object.entries(notifiers).map(([method, stateName]) => {
```

Comment:
we have objectMap for this:
https://github.com/Agoric/agoric-sdk/blob/c3fb5f521f6667c8cbd29dd78910b545a0ca5816/packages/internal/src/utils.js#L76-L82

Reply:
This is pre-existing code and I think I'd like to get rid of this at some point since it messes with types. I'd like to defer that for now though

Reply:
done